docs/openstack/ovn_adoption.md

```diff
@@ -48,13 +48,50 @@ COMPUTE_SSH="ssh -F ~/director_standalone/vagrant_ssh_config vagrant@standalone"
 ```bash
 ${CONTROLLER_SSH} sudo systemctl stop tripleo_ovn_cluster_northd.service
 ```
+- Prepare the OVN DBs copy dir and the adoption helper pod
+
+```yaml
+mkdir -p tmp/ovn
+cat >tmp/ovn/oc_run_args.yaml<<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name: adoption
+spec:
+  containers:
+  - image: $OVSDB_IMAGE
+    command: [ "sh", "-c", "sleep infinity"]
+    name: adoption
+    volumeMounts:
+    - mountPath: /backup
+      name: backup
+  securityContext:
+    allowPrivilegeEscalation: false
+    capabilities:
+      drop: ALL
+    runAsNonRoot: true
+    runAsUser: 1000
+    seccompProfile:
+      type: RuntimeDefault
+  volumes:
+  - hostPath:
```

Comment:
As Sean previously commented, can we avoid using hostPath? Perhaps we should use a PVC like we use for other persistent control plane storage? We should aim for something that would map well to production use.